server/src/test/java/io/crate/integrationtests/MetadataTrackerITest.java

```diff
@@ -297,6 +298,26 @@ public void test_dropped_table_is_replicated() throws Exception {
         });
     }
 
+    @Test
+    public void test_drop_publication_stops_tracking_of_all_other_publications_of_a_subscription() throws Exception {
+        executeOnPublisher("CREATE TABLE t1 (id INT)");
+        executeOnPublisher("INSERT INTO t1 (id) VALUES (1), (2)");
+        createPublication("pub1", false, List.of("t1"));
+
+        executeOnPublisher("CREATE TABLE t2 (id INT)");
+        executeOnPublisher("INSERT INTO t2 (id) VALUES (1), (2)");
+        createPublicationWithoutUser("pub2", false, List.of("t2"));
+
+        createSubscription("sub1", List.of("pub1", "pub2"));
+
+        assertBusy(() -> assertThat(isTrackerActive(), is(true)));
+        var response = executeOnPublisher("SELECT * FROM pg_publication");
+        assertThat(response.rowCount(), CoreMatchers.is(2L));
+
+        executeOnPublisher("DROP PUBLICATION pub1");
+        assertBusy(() -> assertThat(isTrackerActive(), is(false)));
```

Comment:
currently passes, but probably should not - if subscription has many publications and only one publication is dropped, then tracking of other publications also stops